Enhance meta configuration documentation for tests (#8841)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/meta.md

<TabItem value="tests">

Use the `meta` field to add metadata to [generic](/docs/build/data-tests#generic-data-tests) or [singular tests](/docs/build/data-tests#singular-data-tests). `meta` accepts key-value pairs, is compiled into `manifest.json`, and appears in auto-generated documentation.

**Generic data tests**

> Add meta under the config block in your YAML properties file:

Reviewer: ```suggestion
Add `meta` under the `config` block in your `properties.yml` file:
```

<File name="models/properties.yml">
  
```yaml
models:
  - name: my_model
    columns:
      - name: my_column
        data_tests:
          - unique:
              config:
                meta:
                  owner: "docs team"
 ```
</File>

Or set defaults in `dbt_project.yml`:

<File name="dbt_project.yml">

```yaml
data_tests:
  my_project:
    +meta:
      owner: "docs team"
 ```
</File>

**Singular data tests**

> Add meta in the SQL test file using `config()`:

Reviewer: ```suggestion
Add `meta` in the SQL test file using `config()`:
```

<TabItem value="tests">

Use the `meta` field to add metadata to [generic](/docs/build/data-tests#generic-data-tests) or [singular tests](/docs/build/data-tests#singular-data-tests). `meta` accepts key-value pairs, is compiled into `manifest.json`, and appears in auto-generated documentation.

**Generic data tests**

> Add meta under the config block in your YAML properties file:

Author: ```suggestion

Add meta under the config block in your YAML properties file:
```

<File name="models/properties.yml">
  
```yaml
models:
  - name: my_model
    columns:
      - name: my_column
        data_tests:
          - unique:
              config:
                meta:
                  owner: "docs team"
 ```
</File>

Or set defaults in `dbt_project.yml`:

<File name="dbt_project.yml">

```yaml
data_tests:
  my_project:
    +meta:
      owner: "docs team"
 ```
</File>

**Singular data tests**

Add meta in the SQL test file using `config()`:

<File name="tests/my_singular_test.sql">

```sql
{{ config(meta={'owner': 'docs team'}) }}

select * from {{ ref('my_model') }}
where my_column is null
```
</File>

> Or document in `tests/schema.yml`:

Author: ```suggestion
Or document in `tests/properties.yml`:
```

</Tabs>
</VersionBlock>

### Add meta to generic and singular data tests

> This example uses property files for [generic data tests](/docs/build/data-tests#generic-data-tests), and `config()` in SQL for [singular data tests](/docs/build/data-tests#singular-data-tests). You can also set defaults in `dbt_project.yml` or `tests/properties.yml`.

Reviewer: ```suggestion
The following examples show how to add `meta` to [generic data tests](/docs/build/data-tests#generic-data-tests) in a `properties.yml` file, and to [singular data tests](/docs/build/data-tests#singular-data-tests) using `config()`. You can also set defaults in `dbt_project.yml` or `tests/properties.yml`.
```
